xwiki-commons-core/xwiki-commons-velocity/src/main/java/org/xwiki/velocity/tools/EscapeTool.java

```diff
@@ -273,4 +276,14 @@ public String url(Object string)
         }
         return encodedURL;
     }
+
+    /**
+     * @return an instance of escapetool specific for {@code xwiki/21}
+     * @since 14.9RC1
+     */
+    @Unstable
+    public XWiki21EscapeTool getXwiki21()
```

Comment:
typo: it's `XWiki`:)

Reply:
Not if we want to be able to write `$escapetool.xwiki21.parameterArray` in velocity right?

Reply:
Indeed (https://velocity.apache.org/engine/2.0/user-guide.html#property-lookup-rules), `getxwiki21()` would work too but it's not better.

Reply:
Maybe just put a comment to explain it's not a type but that it's for being able to write `$escapetool.xwiki21.`  from Velocity.